src/pages/DonateModal/DonateModal.tsx

```diff
@@ -24,6 +24,9 @@ const DonateModal: React.FC<DonateModalProps> = ({ isVisible, onClose }) => {
     <Modal
       open={isVisible}
       onClose={onClose}
+      style={{ color: '#1498e5' }}
```

Comment:
Shouldn't this be removed? Isn't that what I temporarily added before the dark mode text color fix?

Reply:
Thank you! You're right, I missed that while solving the merge conflict. Fixed 🙏